### PR TITLE
482 column header alerts

### PR DESCRIPTION
**Pull request by kmagida110**
Fixes new issues in #482

### CODE REVIEW

**Review by mmazanec22 — APPROVED** [review]

File: client/src/containers/Upload/Upload.tsx [hunk 1]
```diff
@@ -114,20 +114,22 @@ const Upload: React.FC = () => {
...
-            <div>
-              <p className="margin-bottom-2 text-bold">
-                Download the data collection template
-              </p>
-              <TemplateDownloadLink type="xlsx" />
-              <TemplateDownloadLink type="csv" />
-            </div>
-          }
-        />
+        <div className="margin-bottom-2">
```
Comment (jlhogan) [inline client/src/containers/Upload/Upload.tsx:117]: this is totally fine, but we should make a note to add `className` param to Alert component (ideally all components will take an optional user-provided className prop)

File: src/controllers/enrollmentReports/parse.ts [hunk 2]
```diff
@@ -208,18 +208,26 @@ function getBoolean(value: string): boolean {
  * @param invalidColumns - Array of columns that are invalid
  * @param invalidReason - Single word describing why columns are invalid
  */
+type invalidReasons = 'unmatched' | 'missing';
```
Comment (jlhogan) [inline src/controllers/enrollmentReports/parse.ts:211]: nit: I usually make types title case singular, so `InvalidReason`

**Review by jlhogan — APPROVED** [review]

**Review by JordanGuinn — APPROVED** [review]

**Review by bamader — APPROVED** [review]
LGTM!

File: src/controllers/enrollmentReports/parse.ts (hunk 2 above)
Comment (kmagida110) [reply src/controllers/enrollmentReports/parse.ts]: OK, actually not a nit, I was thinking of it more as an array but it is really an entirely new type

**Comment by kmagida110** [comment]
@RyanHansz Just deployed to QA

**Comment by RyanHansz** [comment]
@kmagida110 not seeing the changes in QA yet

**Pull request merged**